config/cyphal-config.yaml

```diff
@@ -0,0 +1,33 @@
+'11':
```

Comment:
Well, this is a tad suspicious, isn't it? Your formatter appears to have changed integer keys to string keys. Yakut should be able to accept both but still, why would it do that?

Reply:
Good question :thinking:

Reply:
Fixed by https://github.com/107-systems/l3xz/pull/59/commits/258378150e3f644ef03da0d146fa4247a9956eef and using `pyyaml` :bow: .